account_statement_import_online_gocardless/tests/test_account_statement_import_online_gocardless.py

```diff
@@ -80,7 +80,6 @@ def setUpClass(cls):
 
     def test_mocked_gocardless(self):
         vals = {
-            "provider_ids": self.provider.ids,
```

Comment:
Why removing this?

Reply:
Seeing the tests, now I see why you removed it. I was just doing the question for knowing it. Anyway, I see that you were the one removing that field:

https://github.com/OCA/bank-statement-import/commit/bd10c2f59e4c37cc9b9c400bd3eab3a192b257c0#diff-724a1bdf113d58c503e61bbc4e54e4416dc30cc2246214f19bbe2e521539eadeL24

Please remove it again.